services/boundaries/boundaries.go

```diff
@@ -34,3 +35,15 @@ func (svc *service) CampaignsLimitExceeded(user *entities.User) (bool, error) {
 
 	return false, nil
 }
+
+func (s *service) SubscribersLimitExceeded(user *entities.User) (bool, int64, error) {
```

Comment:
Whats the integer for?

Reply:
@gudgl it will be for ImportSubscriber to compare len of csv with limit count.

Reply:
oh i didnt saw that. I am resolving this

Reply:
The same method will check limit and return the actual count it's good to have it here since we will use this method in ImportSubscribers too.

Reply:
If we decide to do it like this, it's good to change it for all methods that check for limits.

Reply:
why? we only have import subscribers why other method need to return the actual count?

Reply:
@FilipNikolovski @gudgl

Reply:
Then change the method name to be more self explainable

Reply:
I think we don't understand each other we should discuss this further on discord channel. @FilipNikolovski @gudgl